frontend/src/metabase/visualizations/lib/map.ts

```diff
@@ -233,7 +240,7 @@ function embedDashboardTileUrl(
   lonField: string,
   parameters?: unknown[],
 ): string {
-  let url = `/api/embed/tiles/dashboard/${token}/dashcard/${dashcardId}/card/${cardId}/${zoom}/${coord.x}/${coord.y}/${latField}/${lonField}`;
+  let url = `/api/${IS_EMBED_PREVIEW ? "preview_" : ""}embed/tiles/dashboard/${token}/dashcard/${dashcardId}/card/${cardId}/${zoom}/${coord.x}/${coord.y}/${latField}/${lonField}`;
```

Comment:
tiny nit: move endpoint into its own variable

```suggestion
  const endpoint = IS_EMBED_PREVIEW ? "preview_embed" : "embed";
  let url = `/api/${endpoint}/tiles/dashboard/${token}/dashcard/${dashcardId}/card/${cardId}/${zoom}/${coord.x}/${coord.y}/${latField}/${lonField}`;
```